src/Components/LandingPage/LandingPage.tsx

```diff
@@ -66,28 +69,10 @@ const LandingPage = ({ handleCheckboxChange }: LandingPageProps) => {
   };
 
   const getLinksForCheckbox = () => {
-    switch (locale) {
-      case 'es':
-        return {
-          privacyPolicyLink: 'https://co.myfriendben.org/es/data-privacy-policy',
-          addTermsConsentToContact: 'https://co.myfriendben.org/es/additional-terms-and-consent-to-contact',
-        };
-      case 'vi':
-        return {
-          privacyPolicyLink: 'https://co.myfriendben.org/vi/data-privacy-policy',
-          addTermsConsentToContact: 'https://co.myfriendben.org/vi/additional-terms-and-consent-to-contact',
-        };
-      case 'fr':
-        return {
-          privacyPolicyLink: 'https://co.myfriendben.org/fr/data-privacy-policy',
-          addTermsConsentToContact: 'https://co.myfriendben.org/fr/additional-terms-and-consent-to-contact',
-        };
-      default:
-        return {
-          privacyPolicyLink: 'https://co.myfriendben.org/en/data-privacy-policy',
-          addTermsConsentToContact: 'https://co.myfriendben.org/en/additional-terms-and-consent-to-contact',
-        };
-    }
+    return {
+      privacyPolicyLink: privacyLink[locale],
+      addTermsConsentToContact: consentToContactLink[locale],
```

Comment:
What happens if you change the language to Russian and click the link?

Reply:
It just toggles the checkbox. I'm going to have the `en-us` links be the fallback. Please hold.